webapp/src/components/expanded_view/reaction_button.tsx

```diff
@@ -66,9 +66,13 @@ export const ReactionButton = forwardRef(({trackEvent}: Props, ref) => {
     const addReactionText = showBar ? 'Close Reactions' : 'Add Reaction';
 
     const handleUserPicksEmoji = (ev: EmojiPickEvent) => {
+        let skin;
+        if (ev.skin) {
+            skin = EMOJI_SKINTONE_MAP.get(ev.skin);
+        }
         const emojiData = {
             name: ev.id,
-            skin: ev.skin ? EMOJI_SKINTONE_MAP.get(ev.skin) : undefined, // eslint-disable-line no-undefined
+            skin,
```

Comment:
Out of curiosity, how is this semantically different than the previous version?

Reply:
@streamer45 It's not; it's technically exactly the same. That's why it's so weird. But I traced the crash to that. It only happened in the global popout (5.3), when you click the emoji icon and the emoji-mart tries to open. It must... I don't know, take the   `handleUserPicksEmoji` that's passed in, and what, run it? It's one of the stranger bugs I've seen, and this fixes it. I still want to know what's going on...

Reply:
Very weird, where you able to reproduce consistently or just randomly?

Reply:
At the time I could reproduce 100% consistently. Now, I can't reproduce at all. 😵‍💫

Reply:
I've tried in 5.2 and 5.3 multiple times, but it looks like whatever the issue was isn't happening anymore.